web/modeshape-webdav/src/main/java/org/modeshape/webdav/methods/AbstractMethod.java

```diff
@@ -116,32 +123,11 @@ protected AbstractMethod() {
     }
 
     public static String lastModifiedDateFormat( final Date date ) {
-        DateFormat df = thLastmodifiedDateFormat.get();
-        if (df == null) {
-            df = new SimpleDateFormat(LAST_MODIFIED_DATE_FORMAT, Locale.US);
-            df.setTimeZone(TimeZone.getTimeZone("GMT"));
-            thLastmodifiedDateFormat.set(df);
-        }
-        return df.format(date);
+        return LAST_MODIFIED_DATE_FORMAT.format(date);
```

Comment:
`SimpleDateFormat` is not thread safe, so you can't share it across threads. The static instance _can_ be used across threads since this method is static, so it's not safe to make this change.